lib/translator/inquiry.cc

```diff
@@ -187,6 +187,109 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+void TranslateLogicalBlockProvisioningVpd(
+    const nvme_defs::IdentifyControllerData &identify_controller_data,
+    const nvme_defs::IdentifyNamespace &identify_namespace_data,
+    absl::Span<uint8_t> buffer) {
```

Comment:
Shouldn't buffer be sent by reference here since we are writing to the buffer we create in the `translate` function below.?

Reply:
span is a wrapper around a pointer so even if its a copy, the span will still wrap around the same pointer